portfolio/src/main/webapp/script.js

```diff
@@ -48,14 +48,105 @@ function showMoreOrLessProjects() {
 }
 
 function getAndPrintComments() {
+
+    /*fetch('/data').then((response) => response.json()).then((comments) => {
     
-    fetch('/data').then((response) => response.json()).then((message) => {
-        var comments = "";
-        for(var i=0; i<message.length; i++){
-            comments+= "<br>"+message[i]+"<hr>";
-            console.log(message[i]);
-        } 
-        document.getElementById('comments-container').innerHTML = "<br><p style='font-size: larger;'><strong><em> Comments: </em></strong></p>"+ comments;
-    });
-    
+    const commentListElement = document.getElementById('comments-container');
+    const buttonWithDropdown= "<div class='dropdown'>"+
+                    "<p><button class='commentSettings' onclick='commentSettings()'><i></i>&#9881;</button></p>"+
+                    "<div id='myDropdown' class='dropdown-content'>"+
+                        "<a href='#' onclick='deleteAllComments()'>Delete all comments</a>"+
+                        "<a href='#'>Set max no. of comments</a>"+
+                    "</div>"+
+                "</div>";
+    var commentsToBeAdded ="";
+    for(const comment of comments) {
+        commentsToBeAdded+= "<br>"+comment;
+    }
+    commentListElement.innerHTML= "<br><p style='font-size: larger;'><strong><em> Comments: </em></strong></p>"+ commentsToBeAdded+ buttonWithDropdown;
+  });*/
```

Comment:
Don't submit commented out code. Code which was already submitted, you can easily get back looking through GitHubs history.

Reply:
Ok, noted!